### ISSUE TITLE
Feature Request: Allow alternative colouring of progress bar indicators

### ISSUE THREAD

**Issue by keeferrourke** [uikit/uikit#2769]
While the `.uk-progress` progress bar is really pretty, it would be nice if the colour of the progress bar could be modified with some other classes such as `.uk-progress-success` which could be applied to a completed progress bar, and `.uk-progress-danger` which could be applied to progress bars which are below a certain threshold value.

This addition would allow one to add visual semantics of good vs. bad values on progress indicators.

These classes could act like the `.uk-button-danger` and similar classes.

**Comment by florianletsch** [comment]
I see your point and I agree this would be useful in some projects. 

Personally, I would prefer to just keep a general notion of "progress" and not add the modifier classes. It is the always the same discussion of how we should balance "essential features" with "bloat" of the UIkit CSS.

However, this is just my personal opinion. I will add this feature request to our internal list to discuss with the team.

**Comment by Uranbold** [comment]
This is really the problem for making a Customizable theme or something. Coloring need to background-color :)

**Comment by keeferrourke** [comment]
@Uranbold, Yes, indeed. The background colour needs to be modified on the progress-bar and progress-value pseudo-elements. In limited testing I found this to be difficult to do without fighting with the existing UI-Kit styles, so I think it would be nice if UI-kit provided native (optional) support for different progress-value colours (danger, warning, default, success).

**Comment by Uranbold** [comment]
Please ;) guys, Consider about this.

Thanks.

**Comment by LoganArnett** [comment]
Has anyone found a work around for this?

**Comment by Uranbold** [comment]
NO :) I have decided to create my own version and waiting for UiKit add for this feature...

**Comment by dugzino** [comment]
There you go, in case anyone still needs a workaround until they implement coloring classes.

```
  .uk-progress-danger { //created class
    &.uk-progress::-webkit-progress-bar { background-color: #f8f8f8; } // this is its background
    &.uk-progress::-webkit-progress-value { background-color: red; } // those are the overflowing color
    &.uk-progress::-moz-progress-bar { background-color: red; }
    &.uk-progress::-ms-fill { background-color: red; }
  }
```

**Comment by dugzino** [comment]
UIkit 3 recently came out of its beta and it's not just a huge refactor based on v2.
So things can be missing hence why we can post feedback.
